src/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To-do-List!!!</title>
+</head>
+<body>
+    <div class="container">
+        <ul id="uList">
+            <li><p>Today's To Do</p></li>
+            <li><input type="text" placeholder="Add to your list..." id="newTask"></li> 
+         </ul>
+         <button type="button" class="clear">Clear all completed</button>
+    </div>
```

Comment:
Kindly add two spaces of indentation. For readability do not use the tab key.
On some code editors, you can set the tab button to do an X amount of spaces.